state.go

```diff
@@ -307,7 +307,7 @@ func (uv *UpdateVerifyState) Handle(ctx *StateContext, c Controller) (State, boo
 		return NewUpdateErrorState(NewTransientError(err), uv.update), false
 	}
 
-	log.Debugf("handle update verify state")
+	log.Info("handle update verify state")
```

Comment:
This is a small work-around for now if we are having no logs at all. I have added other logs to be stored in file, so can change the log level here.